web/src/components/display-dropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { ThIcon } from '@patternfly/react-icons';
+import * as _ from 'lodash';
+
+export type Sizes = 's' | 'm' | 'l';
```

Comment:
thanks! nit: can you just rename to `Size` ? (as a variable of that type has only one size)